drivers/sensor/fxas21002/Kconfig

```diff
@@ -14,6 +14,8 @@ menuconfig FXAS21002
 
 if FXAS21002
 
+if !HAS_DTS_I2C_DEVICE
+
```

Comment:
I can't comment on the proper line, but can you remove the !HAS_DTS_I2C, now that this is wrapped in a !HAS_DTS_I2C_DEVICE